tests/ci/ci_config.py

```diff
@@ -199,13 +202,21 @@ class JobConfig:
     release_only: bool = False
 
 
+# About the "sparse_checkout" option:
+#
+# It's needed only to test the option itself.
+# No particular sense to use it in the build,
+# and it is even slows down the job.
+
+
 @dataclass
 class BuildConfig:
     name: str
     compiler: str
     package_type: Literal["deb", "binary", "fuzzers"]
     additional_pkgs: bool = False
     debug_build: bool = False
+    coverage: bool = False
     sanitizer: str = ""
     tidy: bool = False
```

Comment:
Let's put it close to the field

```suggestion
@dataclass
class BuildConfig:
    name: str
    compiler: str
    package_type: Literal["deb", "binary", "fuzzers"]
    additional_pkgs: bool = False
    debug_build: bool = False
    coverage: bool = False
    sanitizer: str = ""
    tidy: bool = False
    # sparse_checkout is needed only to test the option itself.
    # No particular sense to use it in every build, since it slows down the job.
```